README.md

```diff
@@ -1,2 +1,105 @@
+<p align="center">
+  <a href="https://www.microverse.org/">
+    <img alt="Microverse" src="https://img.shields.io/badge/-Microverse-blueviolet?style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors/Blog_app/blob/dev/LICENSE.md">
+    <img alt="MIT Licensed" src="https://img.shields.io/github/license/Hamzaoutdoors/to-do-list?style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors/Blog_app">
+    <img alt="GitHub last commit (branch)" src="https://img.shields.io/github/last-commit/Hamzaoutdoors/Blog_app/dev?color=blue&style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors/Blog_app">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/Hamzaoutdoors/Blog_app?color=green&label=%E2%98%85%20stars%20&style=flat-square">
+  </a>
+  <a href="https://github.com/Hamzaoutdoors">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/Hamzaoutdoors?color=yellow&logo=github&style=flat-square">
+  </a>
+</p>
+
+<div align="center">
+ <img alt="Microverse" src="https://img.shields.io/badge/Ubuntu-E95420?style=for-the-badge&logo=ubuntu&logoColor=white"> <img src="https://img.shields.io/badge/postgres-%23316192.svg?style=for-the-badge&logo=postgresql&logoColor=white"/>   <img src="https://img.shields.io/badge/ruby-%23CC342D.svg?style=for-the-badge&logo=ruby&logoColor=white"/> <img src="https://img.shields.io/badge/rails-%23CC0000.svg?style=for-the-badge&logo=ruby-on-rails&logoColor=white"/></div>
+
+   </br>
+ <div align="center">
+  <em align="center" style>" Everyday life is like programming, I guess. If you love something you can put beauty into it."</em><strong> - Sabrina Bryan</strong>
+  </div>
+   </br>
+
+# Blog App
+
+A **Blog app** similar classic blog website. With fully functional website that show the list of posts and empower readers to interact with them by adding comments and liking posts.
+ - **User**
+ - **Post**
+ - **Comment**
+ - **Like** 
+
+
+Everything will be based on the ERD class diagram presented below. 
+
+<!-- </br>
+ <div align="center">
+   <kbd>
+    <img alt="gif" src="https://user-images.githubusercontent.com/80895497/143606720-cc2cd9bd-2909-44f0-b2ea-8056741679c4.gif" width=100% />
+    <p>Catalog of my things Console App</p>
+   </kbd>
+</div> -->
+
+</br>
+ <div align="center">
+    <img alt="gif" src="./public/blog_app_erd.png" width=100% />
+    <p>Entity Relationship Diagram (ERD) of Blog App project</p>
+</div>
+
+## Getting Started
+To get a local copy up and running follow these simple example steps :
+
+ - First of all make sure you have both `Ruby` & `Rails` installed in your machine
+ else You can install `Rails` just with this simple command  :
+ `gem install rails`
+ - clone the project with the following command line : 
+```
+$ git clone git@github.com:Hamzaoutdoors/BLog_app.git
+```
+ - Then go to the right folder 
+```
+$ cd 
+```
+ - Run `rails s` and visit http://localhost:3000/  in your browser!
+
```

Comment:
- [ ] I would recommend giving all the necessary steps to run the project locally so that any user will be able to run your project locally.

After clone

```
$ cd yourrepo

$ bundle

$ rails db:create

## When you will add migration file then, you can add command like

$ rails db:migrate

then last command to run the server is 

$ rails s  // which you have already mentioned
```